lib/processCss.js

```diff
@@ -158,18 +161,18 @@ module.exports = function processCss(inputSource, inputMap, options, callback) {
 	var pipeline = postcss([
 		localByDefault({
 			mode: options.mode,
-			rewriteUrl: function(global, url) {
+			rewriteUrl: function(global, origUrl) {
 				if(parserOptions.url){
-					url = url.trim(" ");
+					var url = origUrl.trim(" ");
 
-					if(!loaderUtils.isUrlRequest(url, root)) {
-						return url;
+					if(!origUrl.replace(/\s/g, '').length || !loaderUtils.isUrlRequest(url, root)) {
+						return origUrl;
```

Comment:
Maybe better to `trim()` and have the cleanest code possible + comment  and wait if someone needs the  `\s`  preserved and fix it **when** we have a usecase. I personally doubt that someone will ever need that, but in 3+ months I look at the code and wonder what the fuss was all about 😛